OTLoginServer/Classes/DatabaseManager.cs

```diff
@@ -78,8 +77,12 @@ public ICollection<World> GetWorlds()
                 Name = ConfigLoader.GetString("serverName"),
             };
 
+            world.ExternalAddress = world.ExternalAddressUnprotected = ConfigLoader.GetString("ip");
```

Comment:
you just duplicated what ExternalAddressProtected and ExternalAddressUnprotected  is doing

Reply:
Removed my connecting issue was not related to this..